src/bind.py

```diff
@@ -293,21 +298,20 @@ def _generate_named_conf_local(
             absolute_path=f"{constants.DNS_CONFIG_DIR}/db.{constants.ZONE_SERVICE_NAME}",
             zone_transfer_ips="",
         )
-        for name in zones:
-            if topology is None:
-                pass
-            elif topology.is_current_unit_active:
-                content += constants.NAMED_CONF_PRIMARY_ZONE_DEF_TEMPLATE.format(
-                    name=name,
-                    absolute_path=f"{constants.DNS_CONFIG_DIR}/db.{name}",
-                    zone_transfer_ips=self._bind_config_ip_list(topology.standby_units_ip),
-                )
-            else:
-                content += constants.NAMED_CONF_SECONDARY_ZONE_DEF_TEMPLATE.format(
-                    name=name,
-                    absolute_path=f"{constants.DNS_CONFIG_DIR}/db.{name}",
-                    primary_ip=self._bind_config_ip_list([topology.active_unit_ip]),
-                )
+        if topology is not None:
```

Comment:
Nitpick:
I think if we reverse the return order 
```suggestion
if topology is None:
    return content
...
```
it might help reduce 1 indentation :D